prac_06/guitar_test.py

```diff
@@ -0,0 +1,16 @@
+from prac_06.guitar import Guitar
+
+name = "Gibson L-5 CES"
```

Comment:
The added variables makes it very clear what's being added, but could we just put the names straight into the object declaration?